### ISSUE TITLE
Colors not working proprely

### ISSUE THREAD

**Issue by EliasTheDev** [RaphGL/rofi#10]
I have a problem with my rofi configuration, and I have no idea why is it happening! 

![Rofi](https://user-images.githubusercontent.com/95052783/150851976-c3753702-e77d-41c7-a2a7-44daa50afc35.png)

**Comment by Selion05** [comment]
same here
```
rofi -v
Version: 1.7.2
```
installed with 
```
curl https://raw.githubusercontent.com/dracula/rofi/master/theme/config1.rasi -o ~/.config/rofi/config.rasi
```
I have no other rofi config

**Comment by IT-Luka** [comment]
execute: rofi-theme-selector
select config
enter
alt+a to save

Although, I advise you to use config2.rasi instead, and customize it to your liking. Neither of them are that good, but config2 has decent variables

**Comment by usersink** [comment]
The background colors have been referenced incorrectly. To fix:

1. Search the document for `@background;`
2. Replace all instances with `@background-color;`

**Comment by RaphGL** [comment]
Sorry for the delay, just got back. 
Issue is now fix on [2c9cc1b](https://github.com/dracula/rofi/commit/2c9cc1b0c306465ee3734b8a31b464588edcbb91).
